spec/models/automated_checks/claim_verifiers/qualifications_spec.rb

```diff
@@ -143,7 +143,7 @@ module ClaimVerifiers
             let(:data) do
               {
                 qts_award_date: Date.new(
-                  MathsAndPhysics.first_eligible_qts_award_year.start_year - 1.year,
+                  MathsAndPhysics.first_eligible_qts_award_year.start_year - 1,
```

Comment:
Shouldn't we should be updating all constants with `policy`?

Reply:
@nomasprime Oh I only did that file, silly me. Is it just this extra file or are we talking all instances?
When we have a result with `MathsAndPhysics::<something>` should these become `policy::<something>` too?

Reply:
@nomasprime looks like the qualifications spec is solely focused on M+P currently. So feel its not the appropriate time to do this update.

Reply:
Ah, of course 👍